design/FY2017/MultiplePartialHVACInOneZone.md

```diff
@@ -0,0 +1,289 @@
+
+## Engineering Reference ##
+
+The current section "Zone Equipment Simulation" discusses simulation of air terminal units which are part of an air loop in great detail.  The only statement about other zone equipment is:
+
+"If multiple air-conditioning components are attached to a zone, the components
+are simulated in the order specified by the user assigned priority given in the ZoneHVAC:EquipmentList object."
+
+This section will be expanded to include more details of the ZoneHVAC:EquipmentList simulation, including the accounting for initial zone loads, remaining loads, load to heating setpoint and load to cooling setpoint.
+
+## Example File and Transition Changes ##
+
+Proposed example files:
+
+ - Single-zone residential model with window air conditioner and central system.
+ - Retail store with two rooftop units, one high-efficiency and one low-efficiency serving the same core zone.
+
+Transition will be required for `ZoneHVAC:EquipmentList`.
+
+## Design ##
+
+Not many details worked out here yet.
+
+The logical place to make the load assignements/adjustements is in
+```
+Array1D< Real64 > SequencedOutputRequired;
+Array1D< Real64 > SequencedOutputRequiredToHeatingSP; // load required to meet heating setpoint by sequence
+Array1D< Real64 > SequencedOutputRequiredToCoolingSP; // load required to meet cooling setpoint by sequence
+```
+
+Currently, only the unitary systems use these for control, so other zone equipment will need to be modified to access these, either by passing the desired loads as arguments to the equipment or by passing the equipment list sequence number.
+
+The current functions which set/adjust the sequenced outputs required are:
+```
+ZoneEquipmentManager::InitZoneEquipment (allocates the sequenced output arrays)
+ZoneEquipmentManager::InitSystemOutputRequired (initializes all sequenced output = full output required)
+ZoneEquipmentManager::UpdateSystemOutputRequired (if EquipPriorityNum is passed in, then sets next sequenced output required to be the remaining load)
+ 
+ZoneTempPredictorCorrector::InitZoneAirSetPoints (initiales all sequenced outputs to zero)
+ZoneTempPredictorCorrector::CalPredictedSystemLoad (initializes all sequenced output = full output required)
+```
+
+So, the likely places to implement the load distribution options is in `ZoneEquipmentManager::InitSystemOutputRequired` and `ZoneEquipmentManager::UpdateSystemOutputRequired`.
+
+In order to establish a PLR, some kind of query needs to be done to find out what the equipment can do and if it is active (it could be scheduled off or an airloop or plant loop could be shut down) and how much output it can give.
+
```

Comment:
So a function call in each equipment model which tells you what the operating capacity is. Each equipment model has information on performance curves, availability schedule and plant index and node connections.